packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -106,6 +106,7 @@ const isServerSide = typeof window === 'undefined'
 
 // Recommended settings for PWA-Kit usages.
 // NOTE: they will be applied on both server and client side.
+// retry is always disabled on server side regardless of the value from the options
 const options = {
     queryClientConfig: {
         defaultOptions: {
```

Comment:
If the comment is true and we trust that the changes work as expected:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/3800f33400ecb94313b291500a9a2e1e43ee1b16/packages/template-retail-react-app/app/components/_app-config/index.jsx#L109

I agree that we should remove the `isServerSide` conditional from the default options:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/3800f33400ecb94313b291500a9a2e1e43ee1b16/packages/template-retail-react-app/app/components/_app-config/index.jsx#L117